include/aws/crt/mqtt/MqttClient.h

```diff
@@ -527,6 +552,12 @@ namespace Aws
                     const char *hostName,
                     uint16_t port,
                     const Io::SocketOptions &socketOptions);
+                static void s_connectionInit(
+                    MqttConnection *self,
+                    const char *hostName,
+                    uint16_t port,
+                    const Io::SocketOptions &socketOptions,
+                    aws_mqtt5_client *mqtt5client);
```

Comment:
Trivial: A small addition to Steve's comment about naming `aws_mqtt5_client` parameters: `mqtt5client` here probably should be renamed to `mqtt5CLient`, just for consistency